Hi @mansnils this looks like it doesn't build the micro_allocator_test:

```
tensorflow/lite/micro/micro_allocator_test.cc:836:70: error: too few arguments to function call, expected 3, have 2
      kTfLiteOk, allocator->FinishModelAllocation(model, eval_tensors));
                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                    ^
./tensorflow/lite/micro/testing/micro_test.h:118:15: note: expanded from macro 'TF_LITE_MICRO_EXPECT_EQ'
    auto vy = y;                                                               \
              ^
./tensorflow/lite/micro/micro_allocator.h:137:16: note: 'FinishModelAllocation' declared here
  TfLiteStatus FinishModelAllocation(
               ^
1 error generated.
```

--- a/tensorflow/lite/micro/micro_allocator_test.cc
+++ b/tensorflow/lite/micro/micro_allocator_test.cc
@@ -821,6 +821,7 @@ TF_LITE_MICRO_TEST(TestTypicalFirstOpAndSecondOpWithScratchTensors) {
       /*Only first tensor (t0) is in subgraph input list=*/1);
 
   TfLiteEvalTensor* eval_tensors = nullptr;
+  tflite::ScratchBufferHandle* scratch_buffer_handles = nullptr;
   constexpr size_t arena_size = 4096;
   uint8_t arena[arena_size];
   tflite::MicroAllocator* allocator =
@@ -831,7 +832,8 @@ TF_LITE_MICRO_TEST(TestTypicalFirstOpAndSecondOpWithScratchTensors) {
       allocator->StartModelAllocation(model, op_resolver,
                                       &node_and_registration, &eval_tensors));
   TF_LITE_MICRO_EXPECT_EQ(
-      kTfLiteOk, allocator->FinishModelAllocation(model, eval_tensors));
+      kTfLiteOk, allocator->FinishModelAllocation(model, eval_tensors,
+                                                  &scratch_buffer_handles));
 
   uint8_t* start = eval_tensors[0].data.uint8;
   TF_LITE_MICRO_EXPECT_EQ(0, eval_tensors[0].data.uint8 - start);

